assets/constants.ts

```diff
@@ -6811,6 +6811,24 @@ export default {
         }
       ]
     },
+    "FEEDBACKUPDATES": {
+      "ROUTE": "feedback-updates",
+      "TITLE": "Feedback Updates",
+      "META": [
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "description",
+          // eslint-disable-next-line max-len
```

Comment:
Here and below, use a shorter meta text that fits within the line limit.

Reply:
Done